src/components/MovieShowPage.js

```diff
@@ -1,11 +1,29 @@
 import React from 'react';
+import { connect } from 'react-redux';
+
+const MovieShowPage = props => {
+  const {
+    id,
+    title,
+    backdrop_path,
+    release_date,
+    overview,
+    average_rating
+  } = props;
 
-const MovieShowPage = () => {
   return (
     <div>
-      <h1>MovieShowPage</h1>
+      <img className='backdrop' src={backdrop_path} alt={title} />
```

Comment:
I'm really curious how we could get this displayed as a background image instead (though we can definitely style it to look great as is too!).  I've always put the background image in the css directly so this would be interesting to look into further.